example_configs/tomcat.yml

```diff
@@ -1,33 +1,409 @@
 ---   
+# Example jmx_exporter configuration for use with Tomcat 8.
+
 lowercaseOutputLabelNames: true
 lowercaseOutputName: true
-rules:
```

Comment:
rules need comeback, or  error:
Caused by: while parsing a block mapping
 in 'reader', line 4, column 1:
    lowercaseOutputLabelNames: true
    ^
expected <block end>, but found BlockEntry
 in 'reader', line 12, column 1:
    - pattern: '^Tomcat<type=GlobalR ...